### PR TITLE
new blog

### PR DESCRIPTION
**Pull request by boyd-nguyen**

### CODE REVIEW

File: Australian Digital Observatory/content/blog/a-recap-of-post-api-conference/contents.lr [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+author: Boyd Nguyen
+---
+body:
+
+Last month, we had the opportunity to participate in the [Post-API Conference 2024](https://sites.google.com/umich.edu/postapiconference/home?authuser=0), organised by Prof. Dan Agnus, Prof. Deen Freelon, Associate Professor Jo Lukito, Dr. Jo Lukito, and Dr. John Pasek.
+
+First held in Philadelphia in 2023, the Post-API Conference provides a platform for the research community to learn and share their experience working with social media data. 
+As the name suggests, the conference focuses on acquiring social media data for research in an era where data access is becoming increasingly difficult.
+
+As a research infrastructure facility primarily dealing with social media data, the Digital Observatory has been exploring sustainable non-API approaches to data collection. For that reason, we were invited to talk about our tools and resources in this area.
```
Comment (mctakahashi) [inline Australian Digital Observatory/content/blog/a-recap-of-post-api-conference/contents.lr:12]: should we say 'human data on the web such as social media'?

**Review by mctakahashi — APPROVED** [review]
Hi Boyd, thanks as always for your quick response to my request!  Really appreciate it.  I have read the blog and it looks good.  I just made a minor suggestion.

**Comment by boyd-nguyen** [comment]
@mctakahashi Thanks Marissa! I've made the edit per your feedback. If you are happy with it, could you approve the PR again, please?

**Review by mctakahashi — APPROVED** [review]

**Review by mctakahashi — APPROVED** [review]
done :-)

**Pull request merged**